judo-runtime-core-dao-core/src/main/java/hu/blackbelt/judo/runtime/core/dao/core/processors/DeletePayloadDaoProcessor.java

```diff
@@ -106,6 +105,18 @@ void collectStaments(EClass entityType,
                          Collection<Statement<ID>> statements,
                          InstanceGraph<ID> containerInstanceGraph,
                          EReference container) {
+
+        statmentCollector(entityType, instanceGraph, statements, containerInstanceGraph, container);
+        Collection<InstanceValue<ID>> visited = Sets.newHashSet();
+        checkMandatoryBackReferences(entityType, instanceGraph, statements.stream().filter(s -> s instanceof DeleteStatement<ID>).collect(toSet()), container, visited);
+    }
+
+
+    private void statmentCollector(EClass entityType,
```

Comment:
> statmentCollector

Typo, should be `statementCollector`

Reply:
[0382708](https://github.com/BlackBeltTechnology/judo-runtime-core/pull/201/commits/0382708dba99d21fc499ed64fdba464108421a03)